tracer/test/Datadog.Trace.Tests/Telemetry/config_norm_rules.json

```diff
@@ -468,6 +468,8 @@
   "DD_API_SECURITY_MAX_CONCURRENT_REQUESTS": "api_security_max_concurrent_requests",
   "DD_API_SECURITY_SAMPLE_DELAY": "api_security_sample_delay",
   "DD_API_SECURITY_ENABLED": "api_security_enabled",
+  "DD_API_SECURITY_ENDPOINT_COLLECTION_ENABLED":"api_security_endpoint_collection_enabled",
```

Comment:
I see that https://github.com/DataDog/dd-go/blob/prod/trace/apps/tracer-telemetry-intake/telemetry-payload/static/config_norm_rules.json already contain those values. Good!